src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1508,72 +1509,74 @@ public void setStreamThroughputMbPerSec(int value)
         int oldValue = DatabaseDescriptor.getStreamThroughputOutboundMegabitsPerSec();
```

Comment:
nit: It's too bad this method wasn't initially named `setStreamThroughputMbitPerSec()`. Perhaps it would be best to add `setStreamThroughputMbitPerSec()`, then throw a `@Deprecated` on the old one (and forward so we don't duplicate the logic, etc.)

Reply:
In theory we didn't rework all JMX but I think what you say makes sense here because we have only 5 `DataRate` properties and it looks weird half of the streaming to be with the new methods half not... So maybe for those 2(`setStreamThroughputMbPerSec` and `getStreamThroughputMbPerSec`) I can do it and keep the old format - set by value and not value and unit, WDYT?

Reply:
WFM

Reply:
Did the same for :
   ```
 public void setStreamThroughputMbPerSec(int value);
    public int getStreamThroughputMbPerSec();

    public void setInterDCStreamThroughputMbPerSec(int value);
    public int getInterDCStreamThroughputMbPerSec();
```